### PR TITLE
[vamp-sdk] new package

### PR DESCRIPTION
**Pull request by abique**
Adds vamp-sdk.

### CODE REVIEW

File: ports/vamp-sdk/CONTROL [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Source: vamp-sdk
+Version: 2.9
+Port-Version: 1
```
Comment (autoantwort) [inline ports/vamp-sdk/CONTROL:3]:
```suggestion
```
The first version is implicit version `0`, so you don't need that line :)  
In general, the `vcpkg.json` file format is preferred. So please run `./vcpkg format ports/vamp-sdk/CONTROL` to convert the CONTROL file to a vcpkg.json file :) In the vcpkg.json file you can add `"supports": "!uwp"`. Make sure after your edit the file is properly formatted with `./vcpkg format ports/vamp-sdk/vcpkg.json`

File: ports/vamp-sdk/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+)
+
+vcpkg_install_cmake()
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+
+# # Handle copyright
+file(INSTALL ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/vamp-sdk RENAME copyright)
```
Comment (Cheney-W) [inline ports/vamp-sdk/portfile.cmake:24]:
```suggestion
file(INSTALL ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```

**Review by Cheney-W — CHANGES_REQUESTED** [review]

**Comment by Cheney-W** [comment]
vamp-sdk:x64-uwp and vamp-sdk:arm-uwp build failed, the failure log as below:
[x64-uwp](https://dev.azure.com/vcpkg/_apis/resources/Containers/9060102/failure%20logs%20for%20x64-uwp?itemPath=failure%20logs%20for%20x64-uwp%2Fvamp-sdk%2Finstall-x64-uwp-dbg-out.log)
[arm-uwp](https://dev.azure.com/vcpkg/_apis/resources/Containers/9060102/failure%20logs%20for%20arm-uwp?itemPath=failure%20logs%20for%20arm-uwp%2Fvamp-sdk%2Finstall-arm-uwp-dbg-out.log)
If these failures are expected, please follow autoantwort's suggestion to add "supports": "!uwp" in the vcpkg.json file.

**Comment by abique** [comment]
@autoantwort @Cheney-W Thank you very much for the review.

**Comment by abique** [comment]
Super strange, the failed test was unrelated to vamp-sdk, it seems that it took an other job.

**Comment by Cheney-W** [comment]
osgearth:x64-linux failure is not related with your PR, this is a CI issue, we are working on it.

**Review by Cheney-W — APPROVED** [review]

**Pull request merged**